package.json

```diff
@@ -52,6 +52,7 @@
     "@asyncapi/parser": "^1.1.1",
     "@asyncapi/raml-dt-schema-parser": "^2.0.0",
     "ajv": "^6.10.2",
+    "asyncAPISDK": "file://./../generator-react-sdk",
```

Comment:
I think you can stop using `file://` deps. Use npm link: https://docs.npmjs.com/cli/v6/commands/npm-link.

Reply:
Done 👍